Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -263,32 +261,39 @@ def is_chrome_running_locally(port):
             exp_str = str(exp)
             connection_refused = 'connection refused'
             if connection_refused in exp_str:
-                demisto.debug(f"Failed to connect to Chrome on prot {port} on iteration {i+1}. {connection_refused}")
+                demisto.debug(f"Failed to connect to Chrome on prot {port} on iteration {i + 1}. {connection_refused}")
             else:
                 demisto.debug(
-                    f"Failed to connect to Chrome on port {port} on iteration {i+1}. ConnectionError, {exp_str=}, {exp=}")
+                    f"Failed to connect to Chrome on port {port} on iteration {i + 1}. ConnectionError, {exp_str=}, {exp=}")
 
         # Mild backoff
         time.sleep(DEFAULT_RETRY_WAIT_IN_SECONDS + i * 2)  # pylint: disable=E9003
 
     return None
 
 
-def read_info_file(filename):
+def read_file(filename):
     try:
         with open(filename) as file:
             ret_value = file.read()
             demisto.info(f"File '{filename}' contents: {ret_value}.")
             return ret_value
     except FileNotFoundError:
+        demisto.info(f"File '{filename}' does not exist")
         return None
 
 
-def write_info_file(filename, contents):
+def write_file(filename, contents, overwrite=False):
     demisto.info(f"Saving File '{filename}' with {contents}.")
-    with open(filename, 'w') as file:
-        file.write(f"{contents}")
-        demisto.info(f"File '{filename}' saved successfully with {contents}.")
+    mode = 'w' if overwrite else 'a'
+    try:
+        with open(filename, mode, encoding='utf-8') as file:
+            if not overwrite:
+                file.write("\n")
+            file.write(contents)
```

Comment:
this is ok, but always try to minimize potentially expensive operations (here it's not really expensive but consider this a best practice)
```suggestion
            file.write("\n" if overwrite else "" + contents)
```